coreax/kernel.py

```diff
@@ -368,9 +373,14 @@ def divergence_x_grad_y_elementwise(self, x: ArrayLike, y: ArrayLike) -> Array:
 
 
 class LaplacianKernel(Kernel):
-    """
+    r"""
     Define a Laplacian kernel.
 
+    The Laplacian kernel is defined as
+    :math:`k: \mathbb{R}^d\times \mathbb{R}^d \to \mathbb{R}`,
+    :math:`k(x, y) = output_scale \exp(\frac{||x-y||_1}{2 length_scale^2})`  where
+    :math:`||\cdot||_1` is the :math:L_1`-norm.
+    .
     :param length_scale: Kernel ``length_scale`` to use
```

Comment:
While you're here, please can you max sure all of the kernels (including Linear Kernel) have consistent docstrings for the length_scale and output_scale params.

Reply:
Good point. We'll need to decide if we expect every kernel to have an input and output scale separately (in the current approach it isn't obvious what we expect).